### ISSUE TITLE
Use lang attribute in the language selector

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#4798]
**Describe the problem you would like to solve**
Screen readers use the current language to determine how text should be pronounced. Our language selectors should add a `lang` attribute so that each language is pronounced correctly.

**Describe the solution you'd like**
Use the `lang` attribute as described here: https://twitter.com/romano_cla/status/1130832868039581696?s=19

**Who is asking for this feature?**
Any institution that is concerned about the accessibility of this application.

cc @sssoz

**Comment by NateWr** [comment]
Solved in https://github.com/pkp/pkp-lib/issues/5015
